tasking/serializers/base.py

```diff
@@ -43,23 +44,25 @@ class ContentTypeFieldSerializer(serializers.ModelSerializer):
     Serializer class that provides a contenty_type field and a method
     to validate it
     """
+    target_content_type = serializers.PrimaryKeyRelatedField(
+        many=False, queryset=get_allowed_contenttypes())
 
-    target_type = serializers.CharField(
-        source='target_content_type',
-        allow_blank=False)
-    target_app_label = serializers.CharField(
-        write_only=True,
-        allow_blank=False)
+    # target_type = serializers.CharField(
```

Comment:
With version control you shouldn't be afraid to delete code by commenting it out.
You can always retrieve it.

Reply:
I agree, I think I removed that.